### ISSUE TITLE
Move the closest handle

### ISSUE THREAD

**Issue by teolag** [LeaVerou/multirange#4]
Clicking on the track moves only the one handle. I think it would be more intuitive to move the closest handle.

Example: 
If range is set from 40% to 60%  and i tap at 80% the range should be 40% to 80%
If the initial range is the same and i tap at 20% the final range will be 20% to 60%

**Comment by LeaVerou** [comment]
I agree, but due to the way it's implemented, this would be pretty tricky. :/

**Comment by danielhellier** [comment]
Seems relatively straightforward to do. Cache both input values, calculate which input value is closest to the new value, and change inputs as such.

http://jsbin.com/wodareporo/edit?html,js,console,output
